src/fees/hooks.ts

```diff
@@ -79,6 +80,9 @@ export function useMaxSendAmount(
   const feeEstimate =
     useUsdToTokenAmount(new BigNumber(usdFeeEstimate ?? 0), tokenAddress) ?? new BigNumber(0)
 
+  if (!balance) {
+    return new BigNumber(0)
```

Comment:
why do we need changes to the fee estimation hook for this feature?

Reply:
Yeah, to be honest, I'm a bit concern about this change!
Since this feature, it's possible that we call the `useMaxSendAmount` method with an empty string which throws an error with the old hook. Then I realized the hook was casting a possible `undefined` value, and I thought that wasn't good.
So I think it's ok if this hook returns 0 when the token is invalid instead of crashing the app.

I would really love to avoid calling this hook when we know there is no token, but I can't put the hook under a condition block, right? So I guess there's no other solution than changing the hook? 
Maybe I'm missing an easier solution 😅

Reply:
ah i see. this hook is getting kind of crazy, i think because the use cases are becoming too broad. there is already a place inside the `useEffect` that handles falsy balance, so i was confused about why we need another place. i think i understand the problem, could you provide a default for the balance? like `  const balance = useTokenInfo(tokenAddress)?.balance ?? new BigNumber(0)`

after talking about this, should we also hide the max button if there is no from token selected? up to you, on one hand i don't think the button is helpful but on the other hand idk if it's worth the extra logic because the button won't break 🤷🏻‍♀️